webpack.config.js

```diff
@@ -25,9 +24,9 @@ const EXPLORER_APP = {
     EXPLORER_APP_DOMAIN_REGEX_PROD: '^explorer\\.cow\\.fi|^explorer-prod\\.vercel\\.app',
     EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.explorer\\.cow\\.fi|^explorer-barn\\.vercel\\.app',
 
-    OPERATOR_URL_STAGING_MAINNET: 'https://barn.api.cow.fi/mainnet/api',
-    OPERATOR_URL_STAGING_GOERLI: 'https://barn.api.cow.fi/goerli/api',
-    OPERATOR_URL_STAGING_XDAI: 'https://barn.api.cow.fi/xdai/api',
+    OPERATOR_URL_STAGING_MAINNET: 'https://api.cow.fi/mainnet/api',
+    OPERATOR_URL_STAGING_GOERLI: 'https://api.cow.fi/goerli/api',
+    OPERATOR_URL_STAGING_XDAI: 'https://api.cow.fi/xdai/api',
```

Comment:
Please don't change that.

What you need is in this comment which still applies https://github.com/cowprotocol/explorer/pull/590/files#r1301457728

Reply:
@alfetopito adressed